### PR TITLE
Update to use Weather plugin class

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: pages/AzureTour002.md [hunk 1]
```diff
@@ -70,7 +71,7 @@ Next you will create the application code to gather temperature, humidity, and b
...
+The first thing you need to do is define the objects you will be working with in the application. The three things that matter are the Azure IoT Device object, a Johnny-Five framework object, the Weather Shield plugin object, and an object to represent the Photon. In order to complete this step, you will need the device ID you copied earlier when you were claiming the Photon (or the name/alias you gave the Photon when you updated the firmware to VoodooSpark) and your Particle Cloud access token. To get the access token, open a terminal window (Mac OS X) or Node.js command prompt (Windows) and execute the following command (you may be prompted to login or provide your Particle Cloud password again):
```
Comment (rwaldron) [inline pages/AzureTour002.md:74]: Adds: "the Weather Shield plugin object,"

File: pages/AzureTour002.md [hunk 2]
```diff
@@ -86,29 +87,26 @@ Now add the following code to the __weather.js__ file:
...
 var five = require ("johnny-five");
+var Weather = require("j5-sparkfun-weather-shield")(five);
 var device = require('azure-iot-device');
 var Particle = require("particle-io");
 
 // Set up the access credentials for Particle and Azure
-var particleKey = process.env.PARTICLE_KEY || 'YOUR PARTICLE ACCESS TOKEN HERE';
-var deviceName = process.env.DEVICE_NAME || 'YOUR PARTICLE PHOTON DEVICE ID/ALIAS HERE';
+var token = process.env.PARTICLE_KEY || 'YOUR PARTICLE ACCESS TOKEN HERE';
+var deviceId = process.env.DEVICE_NAME || 'YOUR PARTICLE PHOTON DEVICE ID/ALIAS HERE';
```
Comment (rwaldron) [inline pages/AzureTour002.md:96]: This is purely for name-to-property association :)

File: pages/AzureTour002.md [hunk 3]
```diff
@@ -117,14 +115,15 @@ var client = new device.Client(connectionString, new device.Https());
 
 In this code you define four variables that you will be working with:
 
-1. <code>five</code> - represents the Johnny Five framework capabilities, which provide a type of object model for working with boards like Arduino and Particle.
-2. <code>Particle</code> - represents the Particle IO framework capabilities which exposes the specifics of the Particle boards including the Core and the Photon.
-3. <code>board</code> - a representation of the physical board you are using. This is created by specifying the Particle board and passing in the Particle Access Token and the device ID or alias.
-4. <code>client</code> - This is the agent class that facilitates the communication between your device and the Azure IoT Hub. It takes the device-specific connection string as an argument and establishes a connection with the IoT Hub.
+1. <code>five</code> - represents the Johnny-Five framework capabilities, which provide a type of object model for working with boards like Arduino and Particle.
+2. <code>Weather</code> - represents the Weather Shield plugin, which is compatible with both the Arduino and Photon variants.
```
Comment (rwaldron) [inline pages/AzureTour002.md:119]: This is the only new line added

File: pages/AzureTour002.md [hunk 4]
```diff
@@ -132,62 +131,41 @@ In the following code you will invoke the <code>board.on()</code> function which
...
+      relativeHumidity: this.relativeHumidity,
     });
+
+    // Create the message based on the payload JSON
+    var message = new device.Message(payload);
+    // For debugging purposes, write out the message paylod to the console
+    console.log("Sending message: " + message.getData());
+    // Send the message to Azure IoT Hub
+    client.sendEvent(message, printResultFor('send'));
+  });
```
Comment (rwaldron) [inline pages/AzureTour002.md:168]: Much simpler now! Easier to get to the point :) Less teaching about mechanics of event-based software design and more about reporting IoT data!

File: pages/AzureTour002.md [hunk 5]
```diff
@@ -202,18 +180,17 @@ function printResultFor(op) {
...
+  When you power on the Photon and it establishes a Wi-Fi connection, the first thing it does is a 'phone home' to the Particle Cloud where it registers itself as online. When it does that, it also registers its local IP address. When you run the Node.js application, thanks to the Johnny-Five framework and the Particle-IO plugin, the Node app pings the Particle Cloud and requests the IP address for the device name you specified (that is why the Particle Token and Device ID/Alias are needed). Once the application has the local IP address for the Photon, all communications with the device are over local TCP (which is why your development machine and the Photon have to be on the same network). Since the communication from the Node.js app to the Photon is over TCP and not USB, the Photon doesn't need to be plugged into your USB port - it simply needs to be powered on and on the same Wi-Fi you configured it for (and the machine running the Node.js app has to be on the same Wi-F network).
```
Comment (rwaldron) [inline pages/AzureTour002.md:193]: I'm not sure what this change is, probably some end-of-line whitespace.

File: pages/AzureTour002.md [hunk 6]
```diff
@@ -233,7 +210,7 @@ When you want to quite the application, press <kbd>CTRL</kbd> + <kbd>C</kbd> twi
 
 ## Conclusion &amp; Next Steps
 
-In this lab you learned how to write a Node.js/Johnny Five application that collects environment telemetry and sends it to Azure IoT Hub. In the [next lab][2] you will setup some Azure services to store and visualize the data.
+In this lab you learned how to write a Node.js/Johnny-Five application that collects environment telemetry and sends it to Azure IoT Hub. In the [next lab][2] you will setup some Azure services to store and visualize the data.
```
Comment (rwaldron) [inline pages/AzureTour002.md:213]: Same as https://github.com/ThingLabsIo/ThingLabsIo.github.io/pull/13/files?diff=unified#r41934596

File: pages/AzureTour002.md [hunk 7]
```diff
@@ -241,4 +218,4 @@ In this lab you learned how to write a Node.js/Johnny Five application that coll
 
 [1]: https://store.particle.io/?product=photon-kit
 [2]: /azure/03/
-[deviceexplorer]: https://github.com/Azure/azure-iot-sdks/blob/master/tools/DeviceExplorer/doc/how_to_use_device_explorer.md
\ No newline at end of file
+[deviceexplorer]: https://github.com/Azure/azure-iot-sdks/blob/master/tools/DeviceExplorer/doc/how_to_use_device_explorer.md
```
Comment (rwaldron) [inline pages/AzureTour002.md:221]: Same as https://github.com/ThingLabsIo/ThingLabsIo.github.io/pull/13/files?diff=unified#r41934596

**Comment by dougseven** [comment]
I added these changes in a separate update - there were a couple fixes I had to make to support the Azure bits. Thank you for the Weather Shield plug-in.

**Pull request closed without merging**